### PR TITLE
add launch tech-docs

### PR DESCRIPTION
**Pull request by ahoneybun**
This PR does the following:
- adds the Launch keyboard to the guides page.

### CODE REVIEW

File: content/guides.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ To see ports, keyboard layouts, function keys, and product quickstart guides, fi
 
 | Laptops | Desktops | Accessories |
 |:-------:|:--------:|:-----------:|
-| Adder WS [(addw1)](https://tech-docs.system76.com/models/addw1/README.html) | Meerkat [(meer5)](https://tech-docs.system76.com/models/meer5/README.html) |
+| Adder WS [(addw1)](https://tech-docs.system76.com/models/addw1/README.html) | Meerkat [(meer5)](https://tech-docs.system76.com/models/meer5/README.html) | Launch [(launch_1)](https://tech-docs.system76.com/models/launch_1/repairs.html) |
```
Comment (leviport) [inline content/guides.md:27]: The rest of the models link to `README.html`, not the repairs page. Do you want to link to https://tech-docs.system76.com/models/launch_1/README.html instead?

Comment (storyaddict) [reply content/guides.md:27]: Yes. Lol.

**Review by storyaddict — CHANGES_REQUESTED** [review]
Change link, per feedback from @leviport

**Review by storyaddict — APPROVED** [review]
Article is pointing to the right link now.

**Pull request merged**